app/code/Magento/QuoteGraphQl/Model/Cart/ExtractDataFromAddress.php

```diff
@@ -42,6 +42,7 @@ public function execute(QuoteAddress $address): array
 
         $addressData = array_merge($addressData, [
             'address_id' => $address->getId(),
+            'address_type' => $address->getAddressType(),
```

Comment:
Please, add check to API-functional tests
```
\Magento\GraphQl\Quote\SetShippingAddressOnCartTest
\Magento\GraphQl\Quote\Customer\SetBillingAddressOnCartTest
\Magento\GraphQl\Quote\Guest\SetBillingAddressOnCartTest

Reply:
Tests were updated